scanner/src/routes/q/[iss]/[product]/[id]/+page.svelte

```diff
@@ -0,0 +1,235 @@
+
+  onMount(async () => {
+    data = history.state
+    console.log(data)
+  })
+</script>
+
+<Header />
+<div class="content">
+  <div class="logo-title">
+    <div class="logo"><img src={BatteryPng} alt="" aria-hidden="true" /></div>
+    <div class="title">{data.name}</div>
+  </div>
+  <div class="card">
+    <div class="basic-information">BASIC INFORMATION</div>
+    <div class="row">
+      <div class="property">Verification state:</div>
+      <div class="value">
+        <div class="flex-item">
+          <img src={GreenCheckSvg} alt="" aria-hidden="true" />
+          <span class:trusted={true}>Trusted</span>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="property">product Code:</div>
+      <div class="value">{data.product}</div>
+    </div>
+    <div class="row">
+      <div class="property capitalize">Dataspace domain:</div>
+      <div class="value">{$page.params.iss}</div>
+    </div>
+  </div>
+  <div class="card">
+    <div class="header" class:divider={availableProductFetched}>
+      <div>
+        <div class="dataproduct-name">Available Dataproduct name</div>
+        <div class="subtitle">Data product description</div>
+      </div>
+      <button
+        class="button"
+        class:fetch={!availableProductFetched}
+        class:close={availableProductFetched}
+        on:click={() => (availableProductFetched = !availableProductFetched)}
+      >
+        <span>{availableProductFetched ? "Close" : "Fetch"}</span>
+        <img src={ChevronLeftSvg} alt="" aria-hidden="true" />
+      </button>
+    </div>
+    {#if availableProductFetched}
+      <div class="information">
+        {#each Object.keys(data).filter((key) => !key.includes("svelte")) as key}
+          <div class="row">
+            <div class="property capitalize">{key.replace(/([a-z])([A-Z])/g, "$1 $2")}:</div>
+            <div class="value">
+              {#if data[key] instanceof Array}
+                {#each data[key] as value}
+                  <div class="item">{value}</div>
+                {/each}
+              {:else if typeof data[key] === "object"}
+                {#each Object.keys(data[key]) as nestKey}
+                  <div class="item">
+                    <span class="capitalize">{nestKey.replace(/([a-z])([A-Z])/g, "$1 $2")}</span>
+                    {data[key][nestKey]}
+                  </div>
+                {/each}
+              {:else}
+                {data[key]}
+              {/if}
+            </div>
+          </div>
+        {/each}
+      </div>
+    {/if}
+  </div>
+  <div class="card">
+    <div class="header">
+      <div>
+        <div class="dataproduct-name">Second Dataproduct name</div>
+        <div class="subtitle">Data product description</div>
+      </div>
+      <button class="button unsupport" on:click={() => alert("unsupported")}>Unsupported</button>
+    </div>
+    {#if secondProductFetched}
+      <div class="information" />
+    {/if}
+  </div>
+</div>
```

Comment:
So what part of any of this is coming from the metadata you fetched from the API?

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/8f43646f-46ca-4bbb-af86-c9521fe9d3ea)